core/templates/pages/learner-dashboard-page/new-learner-dashboard/progress-tab.component.html

```diff
@@ -1,24 +1,104 @@
+              Gold
+            </p>
+            <p class="badge-percent">
+              100%
+            </p>
+          </div>
+          <div class="badge-info">
+            <img [src]="silverBadgeImageUrl" alt="silver badge image" class="oppia-badge-image">
+            <p class="badge-type">
+              Silver
+            </p>
+            <p class="badge-percent">
+              80-99%
+            </p>
+          </div>
+          <div class="badge-info">
+            <img [src]="bronzeBadgeImageUrl" alt="bronze badge image" class="oppia-badge-image">
+            <p class="badge-type">
+              Bronze
+            </p>
+            <p class="badge-percent">
+              60-79%
+            </p>
+          </div>
+        </span>
+      </a>
+    </div>
+    <div *ngIf="emptySkillProficiency">
+      <p class="empty-skill-proficiency" [innerHTML]="'I18N_LEARNER_DASHBOARD_EMPTY_SKILL_PROFICIENCY' | translate">
+      </p>
+    </div>
+    <div *ngIf="!emptySkillProficiency">
+      <div *ngFor="let tile of topicsInSkillProficiency; index as i" class="skill-proficiency-content">
+        <div class="skill-proficiency-topic-tile" [ngStyle]="displaySkills[i] && {'border-bottom': 'none', 'margin-bottom': '0'}" *ngIf="tile.practiceTabIsDisplayed">
+          <img [src]="goldBadgeImageUrl" alt="gold badge image" class="oppia-badge-image-for-topic" *ngIf="topicMastery[i] === 100">
+          <img [src]="silverBadgeImageUrl" alt="silver badge image" class="oppia-badge-image-for-topic" *ngIf="topicMastery[i] >= 80 && topicMastery[i] <= 99">
+          <img [src]="bronzeBadgeImageUrl" alt="bronze badge image" class="oppia-badge-image-for-topic" *ngIf="topicMastery[i] >= 60 && topicMastery[i] <= 79">
+          <img [src]="emptyBadgeImageUrl" alt="bronze badge image" class="oppia-badge-image-for-topic" *ngIf="topicMastery[i] > 0 && topicMastery[i] <= 59">
+          <span class="skill-proficiency-topic-title" [ngStyle]="topicMastery[i] === 0 && {'left': '51px'}">
+          {{tile.name}}
+          </span>
+          <img [src]="getStaticImageUrl('/learner_dashboard/dropdown.svg')" alt="dropdown image" class="oppia-svg-dropdown-image" (click)="showSkills(i)" *ngIf="!displaySkills[i]">
+          <img [src]="getStaticImageUrl('/learner_dashboard/up.svg')" alt="up image" class="oppia-svg-up-image" (click)="showSkills(i)" *ngIf="displaySkills[i]">
+          <div *ngIf="topicMastery[i] > 0">
+            <div class="circle-border" [ngStyle]="{'background-image': calculateCircularProgress(i)}">
+              <div class="circle">
+              </div>
+            </div>
+            <span class="progress-text">
+              Progress: {{topicMastery[i]}}%
+            </span>
+          </div>
+        </div>
+        <div class="skill-proficiency-skill-content" *ngIf="displaySkills[i]">
+          <span class="skills-text">
+            Skills
+          </span>
+          <practice-tab [topicName]="tile.name"
```

Comment:
So, the same component in the topic viewer tab is being reused here?

Reply:
Yes.